packages/block-editor/src/store/test/selectors.js

```diff
@@ -2301,6 +2302,16 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( 'getDraggedBlocksOrigin', () => {
```

Comment:
This should be a private selector ("experimental") using new lock/unlock. See note above where I'm unsure as to how to achieve the `unlock()`.

Reply:
✅